changelog/1705.trivial.rst

```diff
@@ -1 +1 @@
-Minor improvements to `~plasmapy.formulary.collisions.CollisionFrequencies`
```

Comment:
Add another changelog feature entry for creating the MaxwellianCollisionFrequencies class!